src/components/CreateVirtualCardsForm.js

```diff
@@ -148,7 +148,7 @@ class CreateVirtualCardsForm extends Component {
         this.setState(state => ({ ...state, values: { ...state.values, numberOfVirtualCards: 1 } }));
       }
     } else if (fieldName === 'amount') {
-      const intAmount = parseInt(value);
+      const intAmount = parseFloat(value);
```

Comment:
@flickz `intAmount` is not a really accurate variable name to hold a value returned by `parseFloat` :wink: 

@znarf I'm personally ok with that, do you agree on allowing float amounts for gift cards values?

Reply:
@Betree I'm ok with it. In the end, might good to have such flexibility.

Reply:
@flickz can you rename `intAmount`?